ui/src/kapacitor/components/config/KafkaConfig.tsx

```diff
@@ -204,6 +234,36 @@ class KafkaConfig extends PureComponent<Props, State> {
     return currentBrokers.map(broker => ({name: broker}))
   }
 
+  private get configID(): string {
+    const {
+      config: {
+        options: {id},
+      },
+    } = this.props
+    return id
+  }
+
+  private get isNewConfig(): boolean {
+    return get(this.props, 'config.isNewConfig', false)
+  }
+
+  private get isDefaultConfig(): boolean {
+    return this.configID === 'default'
```

Comment:
what do you think about factoring out the uses of `default` to be a constant or possibly even a type, to guard against the frailty of string literals?

Reply:
This is the value that we get back from kapacitor so we can't get away from string comparison.